### PR TITLE
[PPP-3735] - Use of vulnerable component httpclient-3.0.1.jar, common…

### PR DESCRIPTION
**Pull request by YuryBY**
…s-httpclient-3.1 - httpclient-4.0.1.jar, SONATYPE-2007-0004, CVE-2012-6153, CVE-2011-1498, CVE-2014-3577

@mbatchelor, @pamval, could you please take a look?

### CODE REVIEW

**Comment by YuryBY** [comment]
@mbatchelor, @pamval, @smaring, @tmorgner could you please take a look?

File: libraries/libpensol/pom.xml [hunk 1]
```diff
@@ -118,6 +118,11 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>pentaho-kettle</groupId>
```
Comment (pamval) [inline libraries/libpensol/pom.xml]: @YuryBY again, bringing in kettle-core as a dependency for libpensol is to be avoided

**Comment by tmorgner** [comment]
@pamval is this safe to merge or are there other dependencies in other projects that need to be merged first? The code itself here looks sane and good enough for a merge.

**Comment by YuryBY** [comment]
Here is rebased PR: https://github.com/pentaho/pentaho-reporting/pull/991

**Pull request closed without merging**